benchmarks/operator_benchmark/pt/qconv_test.py

```diff
@@ -58,6 +70,29 @@
     tags=["resnext101_32x4d"],
 )
 
+class QConv1dBenchmark(op_bench.TorchBenchmarkBase):
```

Comment:
can we factor out the common part of this class so that future support is easier? e.g. we might want conv3d benchmark as well.